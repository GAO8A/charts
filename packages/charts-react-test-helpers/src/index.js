export { default as ChartProvider } from './ChartProvider';
