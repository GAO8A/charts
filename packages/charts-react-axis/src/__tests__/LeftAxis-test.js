import React from 'react';
import renderer from 'react-test-renderer';
import { ChartProvider } from '@ibm-design/charts-react-test-helpers';
import LeftAxis from '../LeftAxis';

describe('LeftAxis Component', () => {
  it('should render', () => {
    const tree = renderer.create(
      <ChartProvider>
        <LeftAxis tickCount={5} />
      </ChartProvider>
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
