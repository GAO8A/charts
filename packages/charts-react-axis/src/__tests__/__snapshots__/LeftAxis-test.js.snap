exports[`LeftAxis Component should render 1`] = `
<g
  className="axis axis--y"
  fill="none"
  fontFamily="sans-serif"
  fontSize="10"
  textAnchor="end">
  <path
    className="domain"
    d="M-6,500H0V0.5H-6"
    stroke="#000"
    style={
      Object {
        "fill": "none",
        "shapeRendering": "crispEdges",
        "strokeWidth": 1,
      }
    } />
  <g
    className="tick"
    transform="translate(0,100)">
    <line
      stroke="#000"
      x2="-6"
      y1="0.5"
      y2="0.5" />
    <text
      dy="0.32em"
      fill="#000"
      x="-9"
      y="0.5">
      Label
    </text>
  </g>
  <g
    className="tick"
    transform="translate(0,200)">
    <line
      stroke="#000"
      x2="-6"
      y1="0.5"
      y2="0.5" />
    <text
      dy="0.32em"
      fill="#000"
      x="-9"
      y="0.5">
      Label
    </text>
  </g>
  <g
    className="tick"
    transform="translate(0,300)">
    <line
      stroke="#000"
      x2="-6"
      y1="0.5"
      y2="0.5" />
    <text
      dy="0.32em"
      fill="#000"
      x="-9"
      y="0.5">
      Label
    </text>
  </g>
  <g
    className="tick"
    transform="translate(0,400)">
    <line
      stroke="#000"
      x2="-6"
      y1="0.5"
      y2="0.5" />
    <text
      dy="0.32em"
      fill="#000"
      x="-9"
      y="0.5">
      Label
    </text>
  </g>
</g>
`;
